src/collective/cover/profiles/testfixture/structure/.objects

```diff
@@ -1,3 +1,4 @@
 my-image,Image
 my-image1,Image
 my-image2,Image
+my-image3,Image
```

Comment:
I'm not sure you need this extra image - and it seems to be breaking the robot tests a bit more than usual!
https://travis-ci.org/collective/collective.cover/jobs/284702469